src/client/training.py

```diff
@@ -11,54 +11,66 @@
 
 @codegen_attr_attributes_setters
 class Training(BaseTolokaObject):
-    """Training pool
+    """A training pool.
 
-    Allows:
-     - Select for the main pool only those Tolokers who successfully complete the training tasks.
-     - Practice Tolokers before the main pool and figure out how to respond correctly.
+    A training pool contains tasks with known solutions and hints for Tolokers. Use training pools:
+    - To train Tolokers so they solve general tasks better.
+    - To select for general pool Tolokers who successfully completed training tasks.
```

Comment:
Sounds just a bit confusing to me, maybe something like
`To select Tolokers who successfully completed training tasks for general pool labeling.`?

Reply:
Eventually changed to: 
To select Tolokers who successfully completed training tasks and to give them access to a general pool.